example_process_video.py

```diff
@@ -13,12 +13,19 @@ def find_videos(search_root):
 
 output_dir = os.getcwd() + "/results"
 
-t = ColorTracker(lower_red, upper_red)
+camera_parameters = None
+if os.path.isfile(os.getcwd() + "/camera_parameters.yaml"):
```

Comment:
We are using `os.getcwd() + "/camera_parameters.yaml"` multiple times in the code, let's extract into local variable.